docs/content/logo.md

```diff
@@ -0,0 +1,191 @@
+            width="250px" />
+            <figcaption>Logo Only</figcaption>
+        </a>
+    </figure>
+    <figure>
+        <a href="https://raw.githubusercontent.com/getporter/community/main/art/icon/porter-icon.png">
+            <img src="https://raw.githubusercontent.com/getporter/community/main/art/icon/porter-icon.png" 
+            alt="Porter Cat's Head: Smiling yellow cat face wearing a red bellhop hat and red bowtie." 
+            width="150px">
+            <figcaption>Profile Picture</figcaption>
+        </a>
+    </figure>
+    <figure>
+        <a href="https://raw.githubusercontent.com/getporter/community/main/art/icon/porter-icon-small.png">
+            <img src="https://raw.githubusercontent.com/getporter/community/main/art/icon/porter-icon-small.png" 
+            alt="Porter Cat as a simplified icon: Smiling yellow cat face that is more angular and less detailed than the normal Porter Cat. She is still wearing the red bellhop hat but does not have the bowtie." 
+            width="64px">
+            <figcaption>Small Icon (<32px)</figcaption>
+        </a>
+    </figure>
+</div>
+
+
+### Wordmark
+
+Porter's wordmark is set in the
+[Abril] typeface:
+
+![Porter Wordmark in the Abril typeface](https://raw.githubusercontent.com/getporter/community/main/art/wordmark/porter-wordmark.png)
+
+Abril is a bold display typeface created by Veronika Burian & José Scaglione (as
+[TypeTogether]). These font files are licensed under the Open Font License.
+
+* [Download Abril Fatface][Abril]
+* [Open Font License][OFL]
+
+Note: For normal page copy please pair a more legible typeface like [Open Sans].
+
+[Abril]: https://fonts.google.com/specimen/Abril+Fatface
+[TypeTogether]: http://www.type-together.com/
+[OFL]: https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL
+[Open Sans]: https://fonts.google.com/specimen/Open+Sans
+
+### Color values: 
+
+![Porter color palette demonstrating the primary brand colors, secondary (shading), and the cat mascot colors](https://raw.githubusercontent.com/getporter/community/main/art/styleguide/porter-palette.png)
+
+Primary (brand elements):
+
+```
+$gray:   #3C2E53; // ideal for text
+$gold:    #DFB960;
+```
+
+Secondary (backgrounds, shading, accents):
+
+```
+$green:  #DDF5F2; // ideal for bg
+$pink:   #CFBFD0;
+```
+
+Cat colors:
+
+```
+$red:     #AB2449;
+$orange:  #F1CE91;
+```
+
+## Porter Cat
+
+
+![Je Suis Porter Cat: Yellow cat standing and waving, wearing a red bellhop hat and bowtie. Underneath is a picture of the cat's face saying "your bundles are packed!". To the side is a repeat of the guidelines below for drawing the Porter Cat mascot](https://github.com/getporter/community/raw/main/art/styleguide/porter-cat-overview.png)
+
+### Guidelines for Drawing Porter Cat
+
+**Simple Shape**
+
+The character should have just enough detail to make them out. The smaller they
+get, the less detail they can have to avoid crowding. The attributes
```

Comment:
"The smaller they get, the less detail they can have" needs some word smithing. Avoid crowding the image by limiting the details as you make the logo smaller.

Reply:
I'm transcribing the text from the images above so that people can read them and make them accessible. For this iteration, is it okay if we go with the original text and later we can work with Ronan to change it? I don't have access to the originals to make the text match if we change the text on the page and it will then be out-of-sync with the picture.